spec/system/candidate_interface/candidate_accepts_offer_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate accepts an offer', sidekiq: true do
+  include CourseOptionHelpers
+
+  scenario 'Candidate views an offer and accepts' do
+    given_i_am_signed_in
+    and_i_have_2_offers_on_my_choices
```

Comment:
nitpick: I noticed we tent to use `_two_` instead of `2`

Reply:
Counterpoint: using the numeral is in the styleguide for government comms:

https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style